net/redis/files/redis.init

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh /etc/rc.common
+
+START=98
+STOP=81
+
+SERVICE_USE_PID=1
```

Comment:
Well, the code should still work but it's NOT actually using procd.

Using procd is more of a personal preference, not a requirement.  It's just that i thought you may want to try procd for good without having to open code these timeout checking etc.